ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md

```diff
@@ -579,6 +579,12 @@ roles/eos_cli_config_gen/docs/tables/management-cvx.md
 roles/eos_cli_config_gen/docs/tables/mcs-client.md
 --8<--
 
+### Metadata
```

Comment:
I think we should move this section to the end of the doc and maybe add a line above the table snip explaining that this data is not generating any config etc.

Reply:
Moved to the bottom with a small paragraph describing the purpose of this variables